src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -105,7 +97,19 @@ class ChaptersButton extends TextTrackButton {
       }));
     }
 
-    if (chaptersTrack) {
+    if (chaptersTrack && !chaptersTrack.cues) {
```

Comment:
Emulated captions should be returning `null` until the track loads. https://github.com/videojs/video.js/blob/0e1c424e48019b002ba542aaceb211b959f511a0/src/js/tracks/text-track.js#L126-L135